modules/ROOT/pages/configuration/server/occ_app_command.adoc

```diff
@@ -795,15 +804,54 @@ Ransomware Protection (Enterprise Edition onyl)
 Marketplace URL: https://marketplace.owncloud.com/apps/ransomware_protection[Ransomware Protection]
 
 Use these commands to help users recover from a Ransomware attack. 
-You can find more information about the application in the documentation <../../enterprise/ransomware-protection/index>.
+You can find more information about the application in the documentation 
+<../../enterprise/ransomware-protection/index>.
 
-[width="100%",cols="50%,82%",]
+....
+sudo -u www-data occ ransomguard:scan <timestamp> <user>
+....
+
+Arguments
+
+[width="100%",cols="20%,70%",]
+|===
```

Comment:
Can you give each of the arguments tables a header. That should make them just that much easier to read — especially in the future if new arguments are added.

Reply:
see above